README.md

```diff
@@ -2,35 +2,33 @@
 
 This project contains custom devcontainers for use in our repositories.
 
-## addons
 
-This devcontainer contains all you need to develop add-ons.
+## Images
 
-Usefull for:
-  - Add-on development
+Image | Description
+-- | --
+`ghcr.io/home-assistant/devcontainer:addons` | For Add-on development
+`ghcr.io/home-assistant/devcontainer:supervisor` | For Supervisor development
```

Comment:
Should we add the path to the Dockerfile those two images are built from? It is probably somewhat obvious, but still doesn't hurt.